testsupport/wait/member.go

```diff
@@ -2313,6 +2314,11 @@ func (a *MemberAwaitility) verifyMutatingWebhookConfig(t *testing.T, ca []byte)
 }
 
 func (a *MemberAwaitility) verifyValidatingWebhookConfig(t *testing.T, ca []byte) {
+	if val := os.Getenv("skip-webhook-checks-on-setup"); val == "true" {
+		// skipped temporarily only for setup migration test but applies for after migration test
+		// This should be removed after PR https://github.com/codeready-toolchain/member-operator/pull/493 is merged
+		return
+	}
```

Comment:
https://github.com/codeready-toolchain/toolchain-e2e/pull/836 is already merged, you can just rebase your branch with the latest changes and drop this workaround

Reply:
Done with latest push